gentoo/gentoo
# sys-cluster/hpx, sci-libs/libgeodecomp: Drop old, fix build issues and add maintainer

None of the current versions of libgeodecomp does compile with any hpx version in tree. That's not good. So

- Drop old versions of all involved libraries
- Fix libgeodecomp to build against hpx
- Close two old hpx bugs (tests and file collision)
- Add mpi support for hpx
- Add myself as maintainer for hpx

If there any objections to this let me know. It should be safe to drop the old hpx version as libgeodecomp is the only user. Furthermore, filed a PR at libgeodecomp to get some of the patches upstream.
